spec/features/user_with_multiple_supplier_can_see_supplier_name_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.feature 'user with multiple suppliers can' do
```

Comment:
This might be a bit picky, but I'd prefer it if these scenarios were each next to their respective scenarios with a single supplier.

Reply:
I've had another crack at these, let me know.